app/Assets/Http/Controllers/AssetManageController.php

```diff
@@ -104,7 +104,7 @@ public function postCreate(CreateRequest $request)
             $resource,
             \Auth::user(),
             \Request::get('name'),
-            \Request::get('description')
+            \Request::get('description') == null ? "" : \Request::get('description')
```

Comment:
You can use the new php null coalesce operator, \Request::get('description') ?? ""